cmd/activator/main.go

```diff
@@ -45,15 +46,18 @@ const (
 )
 
 type activationHandler struct {
-	act    activator.Activator
-	logger *zap.SugaredLogger
+	act      activator.Activator
+	logger   *zap.SugaredLogger
+	reporter activator.StatsReporter
 }
 
 // retryRoundTripper retries on 503's for up to 60 seconds. The reason is there is
 // a small delay for k8s to include the ready IP in service.
 // https://github.com/knative/serving/issues/660#issuecomment-384062553
 type retryRoundTripper struct {
-	logger *zap.SugaredLogger
+	logger   *zap.SugaredLogger
+	reporter activator.StatsReporter
+	start    time.Time
```

Comment:
start inside this struct doesn't feel right. Ideally, we should create one rounttripper instance and keep reusing it.